cmd/cluster-agent/app/app.go

```diff
@@ -129,6 +130,15 @@ func start(cmd *cobra.Command, args []string) error {
 		log.Critical("no API key configured, exiting")
 		return nil
 	}
+	traceAddr := config.Datadog.GetString("agent_host")
+	tracePort := config.Datadog.GetInt("agent_apm_port")
+	traceDebug := config.Datadog.GetBool("tracer_debug_level")
+	if traceAddr != "" {
+		log.Infof("Starting sending traces to %s:%s", traceAddr, tracePort)
+		tracer.Start(tracer.WithAgentAddr(fmt.Sprintf("%s:%d", traceAddr, tracePort)))
+		tracer.WithDebugMode(traceDebug)
```

Comment:
`WithDebugMode` is a `StartOption` and should be passed to `Start` in its variadic args.

Could be something like:
```go
		startOpts := []tracer.StartOption{
			tracer.WithAgentAddr(svc.config.TracerAddr),
		}
		if svc.config.TracerDefaultServiceName != "" {
			startOpts = append(startOpts, tracer.WithServiceName(svc.config.TracerDefaultServiceName))
		}
		tracer.Start(startOpts...)
```